pkg/exchange/ftx/exchange.go

```diff
@@ -179,7 +179,70 @@ func (e *Exchange) QueryAccountBalances(ctx context.Context) (types.BalanceMap,
 	return balances, nil
 }
 
+func (e *Exchange) SupportedInterval() map[types.Interval]int {
+	return map[types.Interval]int{
```

Comment:
pull out to a global ? var supportedIntervals = map[types.Interval]int{....} something like this, and you can reuse them.

Reply:
ok